src/exp/istio-watcher/pkg/watcher/watcher.go

```diff
@@ -39,7 +39,7 @@ Envoy metric sample
 	}
 */
 const (
-	IstioProxyTotalRequestsCMD = "pilot-agent request GET stats?format=json&filter=istio_requests_total"
+	IstioProxyTotalRequestsCMD = "pilot-agent request GET stats?format=json&filter=istio_requests_total\\.*reporter\\.source"
```

Comment:
Isn't it supposed to be formatted with the source workloads name as well ?

Reply:
No, we just filter any metric where the reporter is the source, if you read metrics from the server sidecar, every metric where the server is not the source workload, it will saved as a reporter.destination, and not a source